client/me/profile-links-add-other/index.jsx

```diff
@@ -10,7 +10,8 @@ var FormFieldset = require( 'components/forms/form-fieldset' ),
 	FormTextInput = require( 'components/forms/form-text-input' ),
 	FormButton = require( 'components/forms/form-button' ),
 	eventRecorder = require( 'me/event-recorder' ),
-	SimpleNotice = require( 'notices/simple-notice' );
+	SimpleNotice = require( 'notices/simple-notice' ),
```

Comment:
@mtias is working on refactoring notices in #888. I believe part of that was removing the `SimpleNotice` component. It may be worth checking in on that PR.